.github/workflows/deploy-to-mainnet.yml

```diff
@@ -0,0 +1,36 @@
+name: Deploy Widgets to Mainnet
+on:
+  push:
+    branches:
+      - main
+
+env:
+  ACCOUNT_ID: ${{ vars.MAINNET_ACCOUNT_ID }}
+  ACCOUNT_PUBLIC_KEY: ${{ vars.MAINNET_ACCOUNT_PUBLIC_KEY }}
+  ACCOUNT_PRIVATE_KEY: ${{ secrets.MAINNET_ACCOUNT_PRIVATE_KEY }}
+  NETWORK: mainnet
+
+jobs:
+  deploy-widgets:
+    runs-on: ubuntu-latest
+    name: Deploy widgets to $ACCOUNT_ID
```

Comment:
I feel that $ACCOUNT_ID won't be picked up here. Please, check it on your fork and attach a link with a successful deployment 🙏

Reply:
you're right, it doesn't, I've just removed it
https://github.com/denbite/bos-component-ts-starter/actions/runs/9259579272/job/25471752485